### PR TITLE
feat(atenlib): add ops(normal, narrow)

### PR DESCRIPTION
**Pull request by xiaowuhu**

### CODE REVIEW

File: onnxscript/tests/function_libs/torch_aten/ops_correctness_test.py [hunk 1]
```diff
@@ -455,6 +459,11 @@ def _where_input_wrangler(
         matcher=lambda sample: sample.kwargs.get("as_tuple") is not None,
         reason="as_tuple=True is not supported",
     ),
+    skip(
```
Comment (justinchuby) [inline onnxscript/tests/function_libs/torch_aten/ops_correctness_test.py:462]: What are some other possible types? How can we account for them? Is this a case where the OpInfo does not match the aten op exactly? If that’s the case I would put in reason “non float values for the argument 'mean' are not expected by the aten op”

File: onnxscript/function_libs/torch_aten/ops/core.py [hunk 2]
```diff
@@ -4036,12 +4051,21 @@ def aten_norm_except_dim(v: TensorType, pow: int = 2, dim: int = 0) -> TensorTyp
...
 
+@torch_op("aten::normal")
 def aten_normal(
-    self: TensorType, mean: float = 0.0, std: float = 1.0, generator: Optional[str] = None
-) -> TensorType:
+    self: TTensor,
+    mean: float = 0.0,
+    std: float = 1.0,
+    generator: str = "none",  # pylint: disable=unused-argument
+) -> TTensor:
```
Comment (justinchuby) [inline onnxscript/function_libs/torch_aten/ops/core.py:4060]: Looks like the output type can only be float, according to the doc?

```suggestion
) -> FloatType:
```

File: onnxscript/function_libs/torch_aten/ops/core.py [hunk 3]
```diff
@@ -4036,12 +4051,21 @@ def aten_norm_except_dim(v: TensorType, pow: int = 2, dim: int = 0) -> TensorTyp
...
 
 
+@torch_op("aten::normal")
 def aten_normal(
-    self: TensorType, mean: float = 0.0, std: float = 1.0, generator: Optional[str] = None
-) -> TensorType:
+    self: TTensor,
+    mean: float = 0.0,
+    std: float = 1.0,
+    generator: str = "none",  # pylint: disable=unused-argument
```
Comment (justinchuby) [inline onnxscript/function_libs/torch_aten/ops/core.py:4059]:
```suggestion
```

safe to remocet

File: onnxscript/tests/function_libs/torch_aten/ops_correctness_test.py (hunk 1 above)
Comment (xiaowuhu) [reply onnxscript/tests/function_libs/torch_aten/ops_correctness_test.py:462]: the torch's test case also have Tensor type for 'mean', this is not supported by ort's op.RandomNormalLike().

Comment (xiaowuhu) [reply onnxscript/tests/function_libs/torch_aten/ops_correctness_test.py:462]: ORT only accept float type for args[0] 'mean'

**Review by justinchuby — APPROVED** [review]
thanks!

**Pull request merged**